functions-and-scope.js

```diff
@@ -40,7 +55,17 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat ik alle waardes uit de array kan langslopen, ook als de array wel 100 entries zou bevatten?
 // Log het antwoord in de terminal.
 
+let totalGrades = 0;
+let divide = 0;
```

Comment:
Dit is altijd gelijk aan `grades.length`, dus deze hoef je niet apart bij te houden